pom.xml

```diff
@@ -222,7 +222,7 @@
                                         </goals>
                                     </pluginExecutionFilter>
                                     <action>
-                                        <ignore></ignore>
+                                        <ignore />
```

Comment:
If that plugin does it then we just leave it. If they changed one that was wrong there would likely have been a bug written long ago.  I'm not certain why the Maven parser is so sensitive, it most certainly lies in the XML parser it uses.